lib/core/device.rb

```diff
@@ -683,15 +673,13 @@ def swipe_elements(action)
     el1_y = el1.location.y + (el1.size.height / 2)
     el2_x = el2.location.x + (el2.size.width / 2)
     el2_y = el2.location.y + (el2.size.height / 2)
-    opts = {
-      start_x: el1_x,
-      start_y: el1_y,
-      end_x: el2_x,
-      end_y: el2_y,
-      duration: 2000,
-    }
 
-    action = Appium::TouchAction.new(@driver).swipe(opts).perform
+    @driver.action
+      .move_to_location(el1_x, el1_y)
+      .pointer_down(:left)
+      .move_to_location(el2_x, el2_y, duration: 2)
```

Comment:
why have you put 2 for this and 0.2 for the rest? 2 seconds sounds like a long swipe imo

Reply:
This matches the previously configured value of 2000. All other places were using the default value, which was 200ms.

Reply:
Hmmmmm, I see, well, tbh, I think thats way too long, probably worth it put it 1 second max

Reply:
Okay, so for the sake of keeping the same behaviour as previously, we can keep it like it was. We will have to add an optional value for time in future.